conda/cli/main_config.py

```diff
@@ -283,7 +290,8 @@ def execute_config(args, parser):
             if key not in sequence_parameters:
                 from ..exceptions import CondaValueError
                 raise CondaValueError("Key '%s' is not a known sequence parameter." % key)
-            if not isinstance(rc_config.get(key, []), list):
+            if not (isinstance(rc_config.get(key, []), Sequence) and not 
+                    isinstance(rc_config.get(key, []), str_type)):
```

Comment:
You can change `str_type` to `string_types` here.
`string_types` is already imported from `conda.common.compat` and does the same thing.